bundle/manifests/lcm.openshift.io_imagebasedgroupupgrades.yaml

```diff
@@ -0,0 +1,314 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  creationTimestamp: null
+  name: imagebasedgroupupgrades.lcm.openshift.io
+spec:
+  group: lcm.openshift.io
+  names:
+    kind: ImageBasedGroupUpgrade
+    listKind: ImageBasedGroupUpgradeList
+    plural: imagebasedgroupupgrades
+    shortNames:
+    - ibgu
+    singular: imagebasedgroupupgrade
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ImageBasedGroupUpgrade is the schema for upgrading a group of
+          clusters using IBU
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ImageBasedGroupUpgradeSpec defines the desired state of ImageBasedGroupUpgrade
+            properties:
+              actions:
+                items:
+                  description: ImageBasedUpgradeAction defines the type for the actions
+                    field
+                  enum:
+                  - Abort
+                  - Prep
+                  - Upgrade
+                  - Rollback
+                  - Finalize
```

Comment:
Can we have more verbose description here? Basically would be amazing to know how these map to IBU stage?

Reply:
Sure, I am not sure how to add description here?

Reply:
Ah so i think it's updating the comment [here](https://github.com/openshift-kni/cluster-group-upgrades-operator/pull/780/files#diff-6bb92e0b8f8550f1b369fc7f4a7ac5eb018424f856c6abd502ffea7cb550e1b8R10-R14) and run make generate should do the trick?

Reply:
Oh, I thought you meant add description for each enum item.
updated.